elated-egrets/bangazon-internal-site
# Ry employee detail

## DESCRIPTION

## How to pull down
Commands

```
git fetch --all
git checkout ryEmployeeDetail
```

## Steps to test

1. Start virtual environment
1. Make sure Django server is running
1. click "employee"
1. Click link of each employee details
1. Click back to return to employee list
1. run command: python manage.py test

## What you changed
employee templates
## What was added to read me
nothing
## Final Checklist
Before merging.
1. Has each file changed been reviewed?
2. Has the readme been updated to reflect changes made?
3. Has a teammate approved the PR?

Then Ship It!
